### PR TITLE
Chat and group channel backend

### PR DESCRIPTION
**Pull request by dirtboll**

### CODE REVIEW

File: src/lib/db/tunnel.ts
Comment (ni-tami) [inline src/lib/db/tunnel.ts:1]: disable debugger

**Review by ni-tami — CHANGES_REQUESTED** [review]
Please review my comments

File: src/lib/db/opts.ts
[the remark this answers is not in the document]
Comment (dirtboll) [reply src/lib/db/opts.ts:1]: done

**Review by ni-tami — APPROVED** [review]
LGTM 👏

**Review by ni-tami — APPROVED** [review]
LGTM 👏

**Pull request merged**